Dockerfile

```
@@ -1,71 +1,54 @@
 
-# 3. Production image, copy all the files and run next
-FROM base AS runner
+# ── runner ───────────────────────────────────────────────────────────────────
+FROM node:22-bookworm-slim AS runner
 WORKDIR /app
 
 ENV NODE_ENV=production
+ENV PORT=3000
+ENV HOSTNAME="0.0.0.0"
+
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)